### PR TITLE
update GetVulnerabilityOccurrencesSummary to show fixable/total by

### PR DESCRIPTION
**Pull request by R2wenD2**
severity

### CODE REVIEW

File: v1beta1/proto/grafeas.proto [hunk 1]
```diff
@@ -566,24 +566,18 @@ message GetVulnerabilityOccurrencesSummaryRequest {
 
 // A summary of how many vulnerability occurrences there are per severity type.
```
Comment (vtsao) [inline v1beta1/proto/grafeas.proto]: per resource and severity type.

File: v1beta1/proto/grafeas.proto [hunk 2]
```diff
@@ -566,24 +566,18 @@ message GetVulnerabilityOccurrencesSummaryRequest {
...
-  repeated FixableTotalByDigest fixable_counts = 2;
-  // The number of occurrences for a specific severity.
-  message SeverityCount {
-    // The severity of the occurrences.
-    grafeas.v1beta1.vulnerability.Severity severity = 1;
-    // The number of occurrences with the severity.
-    int64 count = 2;
-  }
-  // Per resource counts of fixable and total vulnerabilites.
+  repeated FixableTotalByDigest fixable_counts = 1;
```
Comment (vtsao) [inline v1beta1/proto/grafeas.proto]: just count now? since we've combined them

Comment (vtsao) [reply v1beta1/proto/grafeas.proto]: or "counts" rather

File: v1beta1/proto/grafeas.proto [hunk 3]
```diff
@@ -566,24 +566,18 @@ message GetVulnerabilityOccurrencesSummaryRequest {
...
+
+  // Per resource and severity counts of fixable and total vulnerabilites.
   message FixableTotalByDigest {
```
Comment (vtsao) [inline v1beta1/proto/grafeas.proto]: Probably should just rename this to "message Counts" in case we decide to change its breakdown later?

Comment (R2wenD2) [reply v1beta1/proto/grafeas.proto]: I'd prefer to leave it more specific. If we change it later we'd need to version anyway.

File: v1beta1/proto/grafeas.proto (hunk 1 above)
Comment (R2wenD2) [reply v1beta1/proto/grafeas.proto]: done

File: v1beta1/proto/grafeas.proto (hunk 2 above)
Comment (R2wenD2) [reply v1beta1/proto/grafeas.proto]: done

**Comment by vtsao** [comment]
Did you commit? I don't see any changes.

**Comment by R2wenD2** [comment]
committed now, I need to fix my tooling.

**Pull request merged**